ppr-ui/src/components/tables/RegistrationTable.vue

```diff
@@ -237,6 +244,7 @@
                     type="text"
                     label="Submitting Party"
                     density="compact"
+                    aria-hidden="true"
```

Comment:
So the A11Y requirements don't want screen readers to pick up the filtering input fields?

Reply:
I will remove this as well. :)

Thanks